### ISSUE TITLE
AWS WAF solution doesn't work in "me-south-1" region

### ISSUE THREAD

**Issue by mogilp** [awslabs/aws-waf-security-automations#120]
From my findings, when try to launch the WAF solution stack in me-south-1 region, it fails with an error :
````
Error occurred while GetObject. S3 Error Code: IllegalLocationConstraintException. S3 Error Message: The unspecified location constraint is incompatible for the region specific endpoint this request was sent to. (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: 1b2a347c-4c83-430a-830c-6db13369ca0e).
````

Please look into this issue further!

**Comment by beomseoklee** [comment]
@mogilp , Thanks for your input.
Unfortunately, we don't currently support ```me-south-1``` region, so if you want to deploy the solution in ```me-south-1``` region, you can follow ```README``` to build and deploy in your account.

Once again, I apologize for your inconvenience, and if you have any other pains while building and deploying by yourself, please let us know.

**Comment by RaviPrasadS** [comment]
@beomseoklee  
aws waf available in me-south-1 region as per latest update 
https://aws.amazon.com/about-aws/whats-new/2020/04/aws-waf-aws-shield-advanced-now-available-hong-kong-bahrain/
But still same error while implementing https://aws.amazon.com/solutions/aws-waf-security-automations/
![AWS-WAF-CF-Error](https://user-images.githubusercontent.com/15628055/78527540-da2fc000-77e5-11ea-8a28-c2012036b928.JPG)

**Comment by RaviPrasadS** [comment]
@beomseoklee 

https://github.com/awslabs/aws-waf-security-automations/issues/127#issuecomment-615429255

@aijunpeng commented on Apr 17 "The reason for the error in me-south-1 region is because the solution isn't officially supported in this new region, but it is on our roadmap. 

But AWS WAF Security Automations Version 2.3.3 last updated: 06/2020 CFT has still same error (attached) while implementing in me-south-1 (Bahrain) region.

kindly check & update on ETA to implement in many customer's aws accounts.

![image](https://user-images.githubusercontent.com/15628055/85225401-978b8600-b3d9-11ea-85d9-70c6e0e5ad9d.png)

**Comment by aijunpeng** [comment]
The new regions haven't been supported by the WAF solution. It is in our backlog for future releases. I don't have an ETA at this time but there will be an announcement when the new regions are supported. Thanks for your patience.

**Comment by maykays** [comment]
This was addressed in V3.1
